src/transformers/models/t5/modeling_tf_t5.py

```diff
@@ -949,6 +949,48 @@ def _shift_right(self, input_ids):
             behaviors between training and evaluation).
 """
 
+T5_ENCODER_INPUTS_DOCSTRING = r"""
+    Args:
+        inputs (:obj:`tf.Tensor` of shape :obj:`(batch_size, sequence_length)`):
+            Indices of input sequence tokens in the vocabulary. T5 is a model with relative position embeddings so you
+            should be able to pad the inputs on the right or the left.
+
+            Indices can be obtained using :class:`~transformers.BertTokenizer`. See
```

Comment:
Should be the T5 tokenizer
```suggestion
            Indices can be obtained using :class:`~transformers.T5Tokenizer`. See
```

Reply:
Great catch!